AGENTS.md

```
@@ -0,0 +1,80 @@
+# Mcode
+
+Performant AI agent orchestration desktop app built with Rust + Tauri.
+
+## Directory Structure
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)